lib/rust/profiler/src/lib.rs

```diff
@@ -746,6 +1013,51 @@ mod tests {
     }
 }
 
+#[cfg(test)]
```

Comment:
It would be nice to have an example/test that shows the JSON output and the result of parsing it. 
Something like serializing some data, asserting it against the string, and deserializing the sample  string and asserting the resulting data.

Reply:
Added a `format_stability` test. In order to avoid the test depending on the exact definitions of event types, it is a deserialize-serialize test rather than a serialize-deserialize test.